metron-deployment/development/centos6/docker_run_ansible.sh

```diff
@@ -14,15 +16,25 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-[defaults]
-host_key_checking = False
-library = ../extra_modules
-roles_path = ../roles
-pipelining = True
-remote_user = centos
-forks = 20
-log_path = ./ansible.log
-
-# fix for "ssh throws 'unix domain socket too long' " problem
-[ssh_connection]
-control_path = ~/.ssh/ansible-ssh-%%h-%%r
+
+
+#
+# This script runs IN the docker container
+#
```

Comment:
Since this script is not intended to be run by the user, what if we put this in a sub-directory?  I think this would make it more clear as to its intended purpose.
```
metron-deployment/development/centos6/docker/run_ansible.sh
````

This would also match how we have all the Ansible stuff living under `metron-deployment/development/centos6/ansible`.

And then there is only one "top-level" script so its blatantly obvious what I'm supposed to run.

Reply:
Good idea, that is how I do it in the plugin, which I did after this, will do